staff/templates/email_templates/new_user_email.html

```diff
@@ -12,4 +12,4 @@ <h4>Welcome to Mission Bit {{ first_name }}</h4>
 <br>
 <br>
 <p>username: {{ username }}</p>
-<p>password: {{ password }}</p>
+<p>random default password: {{ password }}</p>
```

Comment:
`temporary` or `initial` is probably a better term to use than `random default`

Reply:
Done